src/js/_enqueues/wp/widgets/text.js

```diff
@@ -291,7 +291,7 @@ wp.textWidgets = ( function( $ ) {
 				onInit = function() {
 
 					// When a widget is moved in the DOM the dynamically-created TinyMCE iframe will be destroyed and has to be re-built.
-					$( editor.getWin() ).on( 'unload', function() {
+					$( editor.getWin() ).on( 'pagehide', function() {
```

Comment:
The use of the `unload` event may not be a problem here for bfcache since it is added to the iframe and not the window as a whole. Still, if `pagehide` works the same we could still use it.

Reply:
Using `unload` is indeed a problem here. It disabled bfcache:

![image](https://github.com/WordPress/wordpress-develop/assets/134745/7a406fce-5e6b-4316-9e3d-7c82f4a5246c)

Reply:
Removing the handler altogether results in a breakage:

[Screen recording 2023-09-20 17.09.09.webm](https://github.com/WordPress/wordpress-develop/assets/134745/24344ca7-0191-4618-9dde-1ac7edf36f51)

Nevertheless, using `pagehide` instead of `unload` does work:

[Screen recording 2023-09-20 17.09.40.webm](https://github.com/WordPress/wordpress-develop/assets/134745/3bd5e06b-4df2-4295-a159-a45ecb473260)

Reply:
Nevertheless, bfcache remains disabled since TinyMCE is using `unload`. However, this doesn't seem like something we should remove per https://core.trac.wordpress.org/ticket/55491#comment:27